plugin/janitor.py

```diff
@@ -46,12 +48,18 @@ def wrapper_function():
 
 def get_added_lines():
     the_file = vim.eval("expand('%')")
-    tmp_file = os.path.join('/tmp', md5.md5(the_file).hexdigest())
+    with tempfile.NamedTemporaryFile(delete=False) as f:
+        tmp_file = f.name
     vim.command('write! {0}'.format(tmp_file))
 
     cmd = 'diff -u {0} {1}'.format(the_file, tmp_file)
+    startupinfo = subprocess.STARTUPINFO()
```

Comment:
STARTUPINFO() seems not working on non-Windows platforms (I tested on macOS 10.12.5 with ). I got the following error.

```
AttributeError: 'module' object has no attribute 'STARTUPINFO'
```